### PR TITLE
feat: Add `cloudwatch_log_group_arn` to outputs

### PR DESCRIPTION
**Pull request by krrrr38**
# PR o'clock

## Description

- Problem
  - Currently we cannot apply some resources which depends on eks module generated cloudwatch logs **arn**, like followings.
- How to solve?
  - add output `cloudwatch_log_group_arn`

```tf
module "eks" {
  source = "terraform-aws-modules/eks/aws"
  ...
}

resource "aws_lambda_permission" "log" {
  ...
  source_arn    = module.eks.cloudwatch_log_group_arn // <- we cannot pass cloudwatch arn in the same terraform apply.
}
```

To work-around, we need to define data resource. But in this case, we need to execute terraform apply twice to get cloudwatch resource.

```tf
data "aws_cloudwatch_log_group" "log" {
  name = module.eks.cloudwatch_log_group_name[0]
}
```

### Checklist

- [x] CI tests are passing
- [x] README.md has been updated after any changes to variables and outputs. See https://github.com/terraform-aws-modules/terraform-aws-eks/#doc-generation

### CODE REVIEW

File: outputs.tf [hunk 1]
```diff
@@ -61,6 +61,11 @@ output "cloudwatch_log_group_name" {
   value       = aws_cloudwatch_log_group.this[*].name
 }
 
+output "cloudwatch_log_group_arn" {
+  description = "Arn of cloudwatch log group created"
+  value       = aws_cloudwatch_log_group.this[*].arn
```
Comment (barryib) [inline outputs.tf:66]: Can you please return a string instead of a list please. See https://github.com/terraform-aws-modules/terraform-aws-eks/pull/1061

**Review by barryib — CHANGES_REQUESTED** [review]

File: outputs.tf (hunk 1 above)
Comment (krrrr38) [reply outputs.tf]: thanks. updated https://github.com/terraform-aws-modules/terraform-aws-eks/pull/1071/commits/5f61c20b4f03944f04b50d7add5c0f0caa5a72c1

**Comment by krrrr38** [comment]
ci failed on master branch, and fix it in https://github.com/terraform-aws-modules/terraform-aws-eks/pull/1073

**Review by barryib — APPROVED** [review]

**Comment by barryib** [comment]
Thanks @krrrr38 for your contribution.

**Comment by krrrr38** [comment]
than you to maintain this module 🙏

**Pull request merged**